server/models/UserInterests.model.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  AllowNull,
+  Column,
+  CreatedAt,
+  ForeignKey,
+  Model,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript';
+
+interface User_InterestsAttributes {
+  user_id: number;
+  interest_name: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+import Interests from './Interests.model';
+import User from './User.model';
+
+@Table({
+  tableName: 'User_Interests',
+  timestamps: true,
+})
+
+class User_Interests extends Model<User_InterestsAttributes> implements User_InterestsAttributes {
```

Comment:
models should be singular because they describe one thing 
tables should be plural because they house multiple of the same type of things

so for the `User_Interests` model, change it to `User_Interest`
same goes for the file name